cms/djangoapps/contentstore/views/preview.py

```diff
@@ -320,7 +321,8 @@ def _studio_wrap_xblock(xblock, view, frag, context, display_name_only=False):
             'is_selected': context.get('is_selected', False),
             'selectable': context.get('selectable', False),
             'selected_groups_label': selected_groups_label,
-            'can_add': context.get('can_add', True),
+            'can_add': can_add,
+            'can_delete': can_add or (root_xblock and root_xblock.scope_ids.block_type == "itembank"),
```

Comment:
[request] Please add a comment explaining this special-casing of itembank behavior.